pkg/controllers/testing/client.go

```diff
@@ -30,13 +30,17 @@ import (
 	"github.com/projectriff/system/pkg/apis"
 )
 
+type SingularAction interface {
```

Comment:
rm, after other suggestions, this is unused

Reply:
True, nice.